Dockerfile

```diff
@@ -5,7 +5,7 @@ EXPOSE 8000
 
 WORKDIR /
 
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
so we don't actually need to change the aws cli url. in prod, we use linux instance and faris and i agreed on aarch64 for aws cli? @Dashrekker